index.src.html

```diff
@@ -1401,24 +1401,21 @@
 
 - Client processing, and the {{ClientData}} structure, for registration extensions and signature extensions.
 
-- Authenticator processing, and the <a>authenticatorData</a> structure, for signature extensions.
+- Authenticator processing, and the <a>authenticatorData</a> structure, for registration extensions and signature extensions.
```

Comment:
Changed "signature extension" to "authentication extension" since we define registration and authentication to be our two ceremonies. I will open a new issue to add the terms to the glossary, since I am having trouble coming up with polished definitions right now.